SU-SWS/stanford_profile
# Dont remove special characters in person names

# READY FOR REVIEW

# Summary
- Hotfix to prevent special characters from being removed in person names.

# Need Review By (Date)
- asap

# Urgency
- high.

# Steps to Test
1. checkout this branch
1. `drush config-import -y`
1. create a person content and use special characters like dash or & in the first or last name
1. save and verify the special characters are kept.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
